### ISSUE TITLE
Errors in Dataset 

### ISSUE THREAD

**Issue by wangcunxiang** [wangcunxiang/SemEval2020-Task4-Commonsense-Validation-and-Explanation#9]
Thanks for some careful participants, we have collected some errors in the dataset
Since the amount is not too big to affect the results till now, we will not release a new version of the dataset.
Everyone can decide how to use flawed data.
And all participants are welcome to raise new dataset errors here.

**Comment by wangcunxiang** [comment]
1. Unrelated reasons for some statements in TaskB training set (7749, 7750, 7788, 7703, 7811, 7823, 7831, 9605)

**Comment by wangcunxiang** [comment]
2.subtaskB_data_all.csv misses optionC in #5618 #7238 #9941

**Comment by wangcunxiang** [comment]
3. Two sentences are the same in Subtask A training data (4075, 4121, 4314, 6398, 7700)

**Comment by wangcunxiang** [comment]
4. label for 455 in the trial data is wrong

**Comment by piteren** [comment]
> 1. Two sentences are the same in Subtask A training data (4075, 4121, 4314, 6398, 7700)
>    #3

4313, not 4314
